neo/test/iotest/test_stimfitio.py

```diff
@@ -1,33 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I don't think we should remove this unless there are no plans to make stimfitio work with Python 3. The skipped tests are a reminder to fix this.